### ISSUE TITLE
How long does it take to train on two Nvidia Titan RTX GPUs?

### ISSUE THREAD

**Issue by Lightning980729** [chenxin-dlut/TransT#6]
hi, it is a really great job!I has a question?In your paper ,you train on two Nvidia Titan RTX GPUs with the batch size of 38, for a total of1000 epochs with 1000 iterations per epoch.And how long does it take to train in this situation? Thanks.

**Comment by chenxin-dlut** [comment]
In default settings, it takes about 9 days on two Titan RTX GPUs.
In order to save time, you can reduce the number of epochs and decay the learning rate in advance.
For example, train 400 epoch and decay the learning rate at 250th epoch. In this way, the training time will be reduced to about 4 days. According to our experience, it will not have much impact on the performance
